src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIPlugin.groovy

```diff
@@ -39,15 +39,5 @@ class WPIPlugin implements Plugin<Project> {
                 }
             }
         }
-
-        // TODO: Remove for stable, update for 2019 corelibs when appropriate
-        project.afterEvaluate {
-            def style = StyledTextOutput.Style.SuccessHeader
```

Comment:
Should we keep this here, but change it to beta?

Reply:
We can. But since this is 2019 libs were not out of sync anymore.

Reply:
I think a warning along the lines of: 
```
NOTE: You are using a BETA version of GradleRIO, designed for the 2019 Season!
This release requires the 2019 RoboRIO Image, and may be unstable. Do not use this for the official competition season. 
If you encounter any issues and/or bugs, please report them to https://github.com/wpilibsuite/GradleRIO
```

Reply:
Ok, Fixed